utils/docker/dev/Dockerfile

```diff
@@ -10,7 +10,7 @@ ARG GO_VERSION=go1.13
 ARG GO_TAR=$GO_VERSION.linux-amd64.tar.gz
 ARG NANOPB_VERSION=0.3.9.2
 ARG FABRIC_REPO=https://github.com/hyperledger/fabric.git
-ARG FABRIC_VERSION=v1.4.3
+ARG FABRIC_VERSION=1.4.3
```

Comment:
why move the 'v' around?

Reply:
git branches are tagged with v, and docker images are tagged with no v? This is a problem for what?

Reply:
FABRIC_VERSION var is used for both cases you mention: much easier to add the 'v' where needed than to remove it where not